packages/markdown-it-synapse/src/index.ts

```diff
@@ -259,7 +261,7 @@ const synapse: RuleInline = function (state, silent) {
   return true
 }
 
-module.exports = function synapse_plugin(md, _suffix, _baseURL) {
+function synapse_plugin(md, _suffix, _baseURL) {
```

Comment:
For this file, remove commonjs-only syntax like `module.exports`. Let the bundler deal with it.